app/assets/javascripts/short-form/templates/d1-preferences-programs.html.slim

```diff
@@ -10,7 +10,7 @@
       .checkbox-group role="group"
         preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.CERT_OF_PREFERENCE' | translate}}" preference="certOfPreference" labelledby="special_programs_label" dataevent="gtm-preference-cop" translated-description="D1_PREFERENCES_PROGRAMS.CERT_OF_PREFERENCE_DESC"
 
-        preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.DISPLACED' | translate}}" preference="displaced" labelledby="special_programs_label" dataevent="gtm-preference-dthp" translated-description="D1_PREFERENCES_PROGRAMS.DISPLACED_TENANTS_DESC" ng-if="!listingIs('280 Brighton')"
+        preference-without-proof title="{{'D1_PREFERENCES_PROGRAMS.DISPLACED' | translate}}" preference="displaced" labelledby="special_programs_label" dataevent="gtm-preference-dthp" translated-description="D1_PREFERENCES_PROGRAMS.DISPLACED_TENANTS_DESC" ng-show="!listingIs('280 Brighton')"
```

Comment:
yeah it's only because of something to do with all the variables and things going on inside that directive. if it was `ng-if` for a little piece of content it wouldn't matter.